models/PyCryptoBot.py

```diff
@@ -223,6 +225,12 @@ def getSellPercent(self):
         except Exception:
             return 100       
 
+    def getBuyMaxSize(self):
+        try:
+            return int(self.buymaxsize)
```

Comment:
I'm not a Python programmer, but I do not think we need any type cast here. According the code, either the value is None or an positive int (line 170, of the config parser)
I addition, by default `buymaxsize` is set to `None`. So `int(None)` will failed and the function will return None (according the line 232).

I think here we writing the following code is good enough:
``` python
def getBuyMaxSize(self):
   return self.buymaxsize
```

Reply:
You are right. Your code is good enough. I just used the same style/code as other parameters. @whittlem what do you prefer? Change it? or keep it?

Reply:
if you do refer to `getSellPercent` for instance, my comment does apply to that function too.
If the config parser does its job, we should not have to test internally that a member has the proper type.